packages/components/src/components/primitives/form/DropdownField/DropdownNativeSelect/DropdownNativeSelect.test.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import { DropdownNativeSelect, DROPDOWN_NATIVE_DEFAULT_TEST_ID } from '.';
+import { ICON_DEFAULT_TEST_ID } from 'components/primitives/Icon';
+
+describe('DropdownSingleSelect', () => {
+  const options = [
+    { value: 1, label: 'Option 1' },
+    { value: 2, label: 'Option 2' },
+    { value: 3, label: 'Option 3' },
+  ];
+  const onChange = jest.fn();
+  it('should render element with default placeholder', () => {
+    render(<DropdownNativeSelect options={options} onItemChange={onChange} />);
+    screen.getByText('Select an item');
+  });
+
+  it('should render element with placeholder', () => {
+    render(
+      <DropdownNativeSelect
+        options={options}
+        onItemChange={onChange}
+        placeHolder="Placeholder"
+      />
+    );
+    screen.getByText('Placeholder');
+  });
+
```

Comment:
nit: please remove this empty lines between test cases, we are not adding them anywhere else

Reply:
Done!